### ISSUE TITLE
Is there a more descriptive documentation?

### ISSUE THREAD

**Issue by dshacham** [Dhaiwat10/rlp-proxy#14]
I'm trying to create my own proxy and it doesn't seem to work. I don't know what I'm missing. There isn't much documentation to turn to as far as I can see but maybe I'm missing something?

**Comment by Dhaiwat10** [comment]
@dshacham unfortunately this is all the documentation there is. If you could share the problems you are running into, I can have a look and help.

**Comment by dshacham** [comment]
So I managed to make it work. But if I use the customFetcher, whenever I scroll up or down, the card becomes grey for a moment like it's reloading with every scroll. Ever come across something like that?

**Comment by Dhaiwat10** [comment]
Nope. Never experienced that. I think that's an issue with your frontend code rather than the proxy

**Comment by thienbao12a2** [comment]
> So I managed to make it work. But if I use the customFetcher, whenever I scroll up or down, the card becomes grey for a moment like it's reloading with every scroll. Ever come across something like that?

I am having issue setting it up. How did you manage to deploy it?

**Comment by Pnlvfx** [comment]
What is the SERVER_URL variable? is the url where I lunch that script or my express server or what else? Sorry i didn't understand
